src/Doctrine/DBAL/Types/PhoneNumberType.php

```diff
@@ -50,12 +50,17 @@ public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $pla
      */
     public function convertToDatabaseValue($value, AbstractPlatform $platform)
     {
-        if (null === $value) {
+        if (!(bool) $value) {
```

Comment:
This is a wrong change for me. The next error is dedicated to the wrong value format.

Reply:
null is supported by the current code. empty string is a wrong value.